zerver/lib/addressee.py

```diff
@@ -42,8 +42,6 @@ def get_user_profiles_by_ids(user_ids: Iterable[int], realm: Realm) -> List[User
     return user_profiles
 
 def validate_topic(topic: str) -> str:
-    if topic is None:
-        raise JsonableError(_("Missing topic"))
```

Comment:
I kinda want to preserve this error handling; it can probably be an AssertionError though.